### DISCUSSION TITLE
Shared Events

### DISCUSSION THREAD

**Discussion by rsturley** [FamilySearch/GEDCOM#289] [Ideas]
I suggest adding a record type for shared events similar to the SNOTE record type. SEVENT might be a good name for the tag. It would provide a clean way to store shared events, which are already a feature of several genealogy programs. It would have a similar structure as an EVENT, but could be a record with a XREF tag. When used within a FAMIILY_RECORD or INDIVIDUAL_RECORD it would have a cross reference to the SEVENT record and ROLE.

**Comment by tychonievich** [comment]
I'm generally in favor of this, but will comment that in my conversations with the developers at FamilySearch who work with GEDCOM-X, the existence of top-level events is often mentioned as something that trips up developers. We also have anecdotal evidence that SNOTE is itself an implementation challenge: it seems may applications lack a good way to help users understand that they are in fact shared (highlighting that was one of the goals of renaming shared notes SNOTE instead of NOTE like they were in 5.5.1).

>  shared events, which are already a feature of several genealogy programs

I'd be curious to know which ones

**Reply by dthaler** [reply]
Our application (MagiCensus) has census events as top-level custom records, using `_CENS`.

**Reply by cdhorn** [reply]
> I'd be curious to know which ones

Gramps has supported shared events, notes and citations for some years now.

https://gramps-project.org/wiki/index.php?title=Gramps_Data_Model
